src/ai/backend/storage/filebrowser/filebrowser.py

```diff
@@ -0,0 +1,190 @@
+                    },
+                ],
+            },
+            "CpuCount": cpu_count,
+            "Memory": memory,
+            "Cgroup": cgroup,
+            "Mounts": [
+                {
+                    "Target": "/filebrowser_dir/",
+                    "Source": f"{settings_path}",
+                    "Type": "bind",
+                },
+            ],
+        },
+    }
+    for vfolder in vfolders:
+        config["HostConfig"]["Mounts"].append(
+            {
+                "Target": f"/data/{str(vfolder['name'])}",
+                "Source": f"{str(mangle_path(mount_path, vfolder['vfid']))}",
+                "Type": "bind",
+            },
+        )
+    container_name = f"FileBrowser-{service_port}"
+    container = await docker.containers.create_or_replace(
+        config=config,
+        name=container_name,
+    )
+    container_id = container._id
+    await container.start()
+    await docker.close()
+
+    sqlite_db = SQLite_DB(db_path)
+    await sqlite_db.insert_new_container(
+        container_id,
+        container_name,
+        service_ip,
+        service_port,
+        config,
+        "RUNNING",
+        str(datetime.now().strftime("%Y-%m-%d %H:%M:%S")),
+    )
+    return service_ip, service_port, container_id
+
+
+async def recreate_container(container_name, config):
+    try:
+        docker = aiodocker.Docker()
+        container = await docker.containers.create_or_replace(
+            config=config,
+            name=container_name,
+        )
+        await container.start()
+    except Exception:
+        pass
+    finally:
+        await docker.close()
+
+
+async def destroy_container(ctx: Context, container_id: str) -> None:
+    db_path = ctx.local_config["filebrowser"]["db-path"]
+    docker = aiodocker.Docker()
+    sqlite_db = SQLite_DB(db_path)
+    for container in await docker.containers.list():
+        if container._id == container_id:
+            try:
+                await container.stop()
+                await asyncio.sleep(2)
+                await container.delete()
+                await docker.close()
+                await sqlite_db.delete_container_record(container_id)
+                break
+            except Exception:
+                await docker.close()
+    await docker.close()
+
+
+async def get_container_by_id(container_id: str):
+    docker = aiodocker.Docker()
+    container = aiodocker.docker.DockerContainers(docker).container(
+        container_id=container_id,
+    )
+    await docker.close()
+    return container
+
+
+async def get_filebrowsers():
+    docker = aiodocker.Docker()
+    container_list = []
+    containers = await aiodocker.docker.DockerContainers(docker).list()
+    for container in containers:
+        stats = await container.stats(stream=False)
+        name = stats[0]["name"]
+        cnt_id = stats[0]["id"]
+        if "FileBrowser" in name:
+            container_list.append(cnt_id)
```

Comment:
Let's use a container label added when creating the container to distinguish if it's a filebrowser container spawned by us: `ai.backend.container-type` set as `filebrowser`.